src/api/context_models.py

```diff
@@ -1,8 +1,38 @@
 from pydantic import BaseModel
 
 
+class HealthCheckResponseContext(BaseModel):
+    bot_is_avaliable: bool = False
+    site_api_is_avaliable: bool = False
+
+
 class ClosedConsultationContext(BaseModel):
     consultation_id: str
 
     class Config:  # pylint: disable=R0903
         allow_mutation = False
+
+
+class ConsultationContext(BaseModel):
+    consultation_id: str
+    consultation_number: str
+    username_trello: str
+    telegram_id: str
```

Comment:
```suggestion
    telegram_id: int
```
Одной из причин всей этой переделки было "Сделать telegram_id интом" )))